src/commands/publish.js

```diff
@@ -25,27 +25,6 @@ export function toPublishOptions(
 }
 
 async function getUnpublishedPackages(packages) {
-  let semverGtCheckFailWithWarning = (
-    pkgLocalVersion: string,
-    pkgPublishedVersion: string,
-    pkgName: string
-  ): boolean => {
-    let shouldPublish = semver.gt(pkgLocalVersion, pkgPublishedVersion);
-
-    // show a warning message if package is not published since pkgPublishedVersion > pkgLocalVersion
```

Comment:
So mistake here is that the `shouldPublish` check above is correct, but not all cases where we arent publishing are errors (when we are equal for example).